Detectors/ITSMFT/ITS/tracking/GPU/cuda/TimeFrameGPU.cu

```diff
@@ -253,10 +249,12 @@ void TimeFrameGPU<nLayers>::loadTrackingFrameInfoDevice(const int iteration)
 template <int nLayers>
 void TimeFrameGPU<nLayers>::loadMultiplicityCutMask(const int iteration)
 {
-  if (!iteration) {
+  if (!iteration || iteration == 3) { // we need to re-load the swapped mult-mask in upc iteration
```

Comment:
@mconcas This and moving the wiping of the TF data after processing fixed the upc iteration.